src/components/home/modals/AliasAddress.vue

```diff
@@ -0,0 +1,313 @@
+      </b-form-group>
+      <b-form-group label="Alias Name"
+                    label-for="alias-input">
+        <b-form-input id="alias-input"
+                      class="amount-input"
+                      v-model="curAlias"
+                      :readonly="from3rdParty"
+                      aria-describedby="inputLiveFeedback"
+                      :state="isValidAlias"
+                      onfocus="this.select()">
+        </b-form-input>
+        <b-form-invalid-feedback id="inputLiveFeedback"
+                                 v-if="!isAliasLessEight">
+          Please enter an alias no more than 8 characters.
+        </b-form-invalid-feedback>
+        <b-form-invalid-feedback id="inputLiveFeedback"
+                                 v-if="isExistedAlias">
+          You have set this alias to another address, please change it to another！
+        </b-form-invalid-feedback>
+      </b-form-group>
+      <b-row>
+        <b-col class="col-lef">
+          <b-button
+            class="btn-cancel"
+            block
+            variant="light"
+            size="lg"
+            @click="closeModal">Close
+          </b-button>
+        </b-col>
+        <b-col class="col-rit">
+          <b-button
+            block
+            class="btn-add"
+            variant="warning"
+            size="lg"
+            @click="addAlias">Add
+          </b-button>
+        </b-col>
+      </b-row>
+    </div>
+  </b-modal>
+</template>
+
+<script>
+import Vue from 'vue'
+export default {
+    name: 'AliasAddress',
+    props: {
+        addresses: {
+            type: Object,
+            require: true,
+            default: function() {}
+        },
+        coldAddresses: {
+            type: Object,
+            require: true,
+            default: function() {}
+        },
+        address: {
+            type: String,
+            require: true,
+            default: ''
+        }
+    },
+    data() {
+        return {
+            curAlias: '',
+            curAddress: '',
+            aliasNum: 3,
+            aliasAddresses: {}
+        }
+    },
+    created() {
+        if (JSON.parse(window.localStorage.getItem(this.address)) != null) {
+            this.aliasAddresses = JSON.parse(window.localStorage.getItem(this.address)).alias
+        }
+    },
+    computed: {
+        isValidAddress() {
+            if (!this.curAddress) {
+                return void 0
+            }
+            return this.isExistedAddress
+        },
+        isExistedAddress() {
+            return this.curAddress in this.addresses || this.curAddress in this.coldAddresses
+        },
+        isExistedAlias() {
+            if (JSON.parse(window.localStorage.getItem(this.address)) != null) {
+                let alias = JSON.parse(window.localStorage.getItem(this.address)).alias
+                if (alias) {
+                    for (let key in alias) {
+                        if (this.curAlias === alias[key]) {
```

Comment:
Change alias match with case insensitive (save and display alias with upper and lowercase letters, but match alias with lowercase)